CHANGELOG.md

```diff
@@ -16,6 +16,8 @@
 
 ## Unreleased
 
+- Bug fix in aws_iam_authorization to utilize correct secret from env key name ([PR #1332](https://github.com/kedacore/keda/pull/1332))
+
 ### New
 
 ### Improvements
```

Comment:
Could you please move the entry here?

Reply:
Could you let me know what exactly changed? :) I am not seeing a difference but might be overlooking it

Reply:
To move the line, that you have added under the `### Improvements` section.

Reply:
Aha! Right, sorry was missing the correct section 😊

Reply:
No worries, thanks!